core/pyos.py

```diff
@@ -282,6 +282,14 @@ def InputAvailable(fd):
     return len(r) != 0
 
 
+def WaitForInputs(fd):
+    # type: (List[int]) -> List[int]
+    # Get a list of fds and return fds which have available input
+    # read, write, except
+    r, w, exc = select.select(fd, [], fd)
```

Comment:
Hm I want to see what we should do about `exc`

Well at least we can see what other programs / shells do

from `man select`, I guess none of thse apply since it is a pipe ...  well that could be in a comment here



```
       POLLPRI
              There is some exceptional condition on the file descriptor.  Possibilities include:

              •  There is out-of-band data on a TCP socket (see tcp(7)).

              •  A pseudoterminal master in packet mode has seen a state change on the slave (see ioctl_tty(2)).

              •  A cgroup.events file has been modified (see cgroups(7)).
```

Reply:
If we're not going to use `w` and `exc`, then we can call them `_, _`

Also, if we ignore `exc`, then I guess we should not pass the third param to select